example/ck_tile/01_fmha/codegen/ops/fmha_fwd.py

```diff
@@ -485,15 +488,13 @@ def get_pipelines(dtype, hdim) -> List[FmhaFwdPipeline]:
         if d == None:
             continue
         #for hdim_str, mode, mask, bias, lse in itertools.product(d.keys(), MODE_MAP.keys(), MASK_MAP.keys(), ["t", "f"], ["t", "f"]):
-        for hdim_str, mode in itertools.product(d.keys(), MODE_MAP.keys()):
-            tile = d[hdim_str]
-            hdim = int(hdim_str)
-            for pipeline in get_pipelines(dtype, hdim):
+        for ((hdim, hdim_v), tile), mode in itertools.product(d.items(), MODE_MAP.keys()):
+            for pipeline in get_pipelines(dtype, hdim, hdim_v):
                 if mode == "group":
                     if pipeline.F_spad != 't' or pipeline.F_skpad != 't':
                         # in group mode, spad/skpad must be true, since we can't predict if seqlen of current batch need pad or not
                         continue
-                if hdim == 192 and tile.F_bn1 == 128:
+                if hdim in (96, 160, 192):
                     # NOTE: this is used to speedup deepseek prefill case, we don't gen training
```

Comment:
this code here is originally to pattern match hdim=192, hdim_v=128 case, not generate `bias`, `lse`, `dropout`. But this line change the logic to all the potential instance with hdim=96/160/192 case. Either this need not change the original logic, or need think about how to deal with this situation more systematically

Reply:
Agree that we need to do instantiation better. Updated to keep the previous behavior for `hdim=192, hdim_v=128` and ignore `hdim=160` case as decided for now.